internal/wire/transport_parameters.go

```diff
@@ -47,6 +48,8 @@ const (
 	maxDatagramFrameSizeParameterID transportParameterID = 0x20
 	// https://datatracker.ietf.org/doc/draft-ietf-quic-reliable-stream-reset/06/
 	resetStreamAtParameterID transportParameterID = 0x17f7586d2cb571
+	// Multipath transport parameters (draft-ietf-quic-multipath)
+	transportParameterIDInitialMaxPathID transportParameterID = 0xba00 // Experimental codepoint
```

Comment:
You would think that AI would at least be able to use the correct value for the transport parameter (0x0f739bbc1b666d0d): https://www.ietf.org/archive/id/draft-ietf-quic-multipath-14.html#name-initial_max_path_id-transpo